docs/processes/quality-assurance/templates/test-strategy-template.md

```diff
@@ -0,0 +1,54 @@
+## **Test Strategy (Sample)**
+
+**Introduction**
+
+A high-level summary of the project
+
+**Example**
```

Comment:
**suggestion:** Small typo. `Examples`?

Reply:
Hey Tan. Thanks for all the feedback. For this particular one, this subheading refers to this being  *an* example of *an* introduction, so I believe Example is correct here :)